edx_rest_api_client/client.py

```diff
@@ -34,5 +34,6 @@ def __init__(self, url, signing_key=None, username=None, full_name=None, email=N
         super(EdxRestApiClient, self).__init__(
             url,
             session=session,
-            auth=auth
+            auth=auth,
+            append_slash=kwargs.get('append_slash', True),
```

Comment:
i suggest we just pass `**kwargs` along here.  that way next time there's another slumber feature we want to take advantage of, no change will be needed.